From a02bb3bf199a70d7aae13641ae4fcf391001bc29 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 19 Jun 2017 13:05:23 +0800
Subject: [PATCH 3024/5242] MLK-15110-4 gpu: imx: dpu: Select IMX8_PRG and
 IMX8_DPRC in Kconfig

commit  0ee8bc66e765b14f1c4aa284233a4bb839c05d8e from
https://source.codeaurora.org/external/imx/linux-imx.git

Since we needs to use PRG and DPR in DPU base driver, let's select
the two drivers when we enable DPU.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/Kconfig |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/imx/dpu/Kconfig b/drivers/gpu/imx/dpu/Kconfig
index d5f0e55..3c680bb 100644
--- a/drivers/gpu/imx/dpu/Kconfig
+++ b/drivers/gpu/imx/dpu/Kconfig
@@ -3,6 +3,8 @@ config IMX_DPU_CORE
 	depends on ARCH_FSL_IMX8QM || ARCH_FSL_IMX8QXP
 	select RESET_CONTROLLER
 	select GENERIC_IRQ_CHIP
+	select IMX8_PRG
+	select IMX8_DPRC
 	help
 	  Choose this if you have a Freescale i.MX8QM or i.MX8QXP system and
 	  want to use the Display Processing Unit. This option only enables
-- 
1.7.9.5

